_data/edu.yml

```diff
@@ -1,85 +1,148 @@
-  para: You are not restricted by any EULA, and have full access to the source code.
-        Minetest is available under the LGPLv2.1+ license.
-
-- title: Runs well on old hardware
-  para: Minetest is written in a computer language that makes more efficient use of hardware,
-        allowing more schools to be able to experience interactive virtual learning.
-
-- title: Independent
-  para: Minetest is not associated with any company or commercial entity, which can make it
-        more suitable for adoption in organisations where this is a problem.
-
-- title: Friendly Community
-  para: You can find help with any problems on our <a href="/get-involved/">forums or IRC channels.</a>
-
-
-featured:
-- title: Framinetest
-  author: Framasoft
-  lang: fr
-  img: framinetest.png
-  para: An educational server, with geology, biology, and historical reconstruction to imagine your world!
-  url: https://framinetest.org
+resources:
 
-- title: QiskitBlocks
-  author: James Weaver
-  img: qiskitblocks.png
-  para: Teaching Quantum Computing using Qiskit in a block world.
-  url: https://content.minetest.net/packages/JavaFXpert/qiskitblocks/
+- title: Mining for Education
+  author: Paul Brown, OS Mag
+  para: >-
+    An article explaining why you should use Minetest for education, and a guide
+    in doing so.
+  url: https://www.ocsmag.com/mining-for-education/
 
-- title: Craft the Web
-  author: CERN
-  tag: Unreleased
-  img: ctw.png
-  para: Learn what makes the Web function, and about the ideas and major inventions that paved its creation.
-  url: https://forum.minetest.net/viewtopic.php?t=22871
+- title: Learning with Minetest
+  author: Teacher Squeaks
+  para: >-
+    A 3-part post-mortem based on the author's experience of using Minetest in a
+    pre-school.
+  url: http://teachersqueaks.blogspot.co.za/2016/08/minewhat-rolling-out-minetest-in-pre.html
 
+- title: Build Block Worlds with Minetest
+  author: Miguel Guhlin
+  para: ""
+  url: https://blog.tcea.org/build-block-worlds-with-minetest/
 
-resources:
+- title: Minetest Wiki (Edu page)
+  author: The official Minetest Wiki, by the community
+  para: >-
+    A trove of information, in multiple languages, with a space dedicated to
+    education. Looking for more contributions!
+  url: https://wiki.minetest.net/MinetestEDU
 
-- title: BLOCKALOT
-  author: Landesmedienzentrum Baden-Württemberg
-  lang: de
-  img: imvec.png
-  para: >
-    A dashboard to create worlds for future-oriented learning without hosting
-    servers. Comes with support (in English/French/German), a wiki and
-    workshops. It's currently free to use.
+- title: ContentDB
+  author: To find Minetest mods
+  para: >-
+    A database where modders can submit their mods to make them easily
+    accessible. Includes features such as a tag system, reviews, and more.
+  url:  https://content.minetest.net/packages/?tag=education
+
+- title: BLOCKALOT Server Hosting
+  author: To host an Edu server
+  para: >-
```

Comment:
The BLOCKALOT staff provided the previous description and insisted on "future-oriented learning" being an important part of the description. So might be worth reverting this description change